src/Microsoft.AspNetCore.SignalR.Core/Protocol/SerializedMessage.cs

```diff
@@ -3,7 +3,7 @@
 
 using System;
 
-namespace Microsoft.AspNetCore.SignalR
+namespace Microsoft.AspNetCore.SignalR.Protocol
```

Comment:
Why is this in protocol?

Reply:
All the other message classes are in `Microsoft.AspNetCore.SignalR.Protocol`. It feels weird that these "internalish only" infrastructure classes for messages are in the root SignalR namespace.

Reply:
This isn't a message type. It's sits above all protocols and that's why it's in the root. There's a public WriteAsync on HubConnectionContext that takes this in.